examples/matter-controller/BUILD.gn

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+import("${build_root}/config/android_abi.gni")
+import("${chip_root}/build/chip/java/rules.gni")
+import("${chip_root}/build/chip/tools.gni")
+
+android_binary("matter-controller") {
```

Comment:
"matter-controller" is an extremely generic name for "Java CLI Controller"...  Please use "java-matter-controller" or "java-matter-tool" or something along those lines.

Reply:
If we follow examples, it should probably be one of `examples/matter-controller/java` or `examples/java-matter-controller`.

Given that we already have `darwin-framework-tool` maybe the latter is better. At some point we may want to unify naming to have a consistent name for all of 'framework tool, matter controller and chip tool'.

Reply:
The name 'java' should certainly be in there. That's my only requirement.

`java-matter-controller` would be fine by me, but it's a mouthful...

Reply:
Lets use examples/java-matter-controller